extensions/inference-nitro-extension/resources/models/llama3-8b-instruct/model.json

```
@@ -0,0 +1,34 @@
+{
+    "sources": [
+      {
+        "filename": "Meta-Llama-3-8B-Instruct-Q4_K_M.gguf",
+        "url": "https://huggingface.co/lmstudio-community/Meta-Llama-3-8B-Instruct-GGUF/resolve/main/Meta-Llama-3-8B-Instruct-Q4_K_M.gguf"
+      }
+    ],
+    "id": "llama3-8b-instruct",
+    "object": "model",
+    "name": "Llama 3 8B Q4",
+    "version": "1.0",
+    "description": "The latest model from MetaAI",
```

```suggestion
    "description": "Meta's Llama 3 excels at general usage situations, including chat, general world knowledge, and coding.",
```
More descriptive option, derived from the huggingface model card that the dowload URL points to (lmstudio). Being the "latest" model does not age well at the current speed of developments